internal/delivery/http/handlers/feedbacks.go

```diff
@@ -0,0 +1,209 @@
+func (h *Handlers) GetFeedback(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	feedbackID := chi.URLParam(r, "id")
+	if feedbackID == "" {
+		h.handleError(w, http.StatusBadRequest, errIDParamIsMissing)
+
+		return
+	}
+
+	feedback, err := h.feedbackService.GetByID(feedbackID)
+	if err != nil {
+		h.handleError(w, http.StatusNotFound, err)
+
+		return
+	}
+
+	err = json.NewEncoder(w).Encode(feedback)
+	if err != nil {
+		h.handleError(w, http.StatusInternalServerError, err)
+
+		return
+	}
+}
+
+// GetAllFeedback GET /feedbacks.
+func (h *Handlers) GetAllFeedback(w http.ResponseWriter, _ *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	feedbacks, err := h.feedbackService.GetAll()
+	if err != nil {
+		h.handleError(w, http.StatusBadRequest, err)
+
+		return
+	}
+
+	err = json.NewEncoder(w).Encode(feedbacks)
+	if err != nil {
+		h.handleError(w, http.StatusInternalServerError, err)
+
+		return
+	}
+}
+
+// CreateFeedback POST /feedback.
+func (h *Handlers) CreateFeedback(w http.ResponseWriter, r *http.Request) {
+	var feedback models.FeedbackInput
+
+	w.Header().Set("Content-Type", "application/json")
+
+	err := json.NewDecoder(r.Body).Decode(&feedback)
+	if err != nil {
+		h.handleError(w, http.StatusBadRequest, err)
+
+		return
+	}
+
+	feedbackID, err := h.feedbackService.Create(&feedback)
+	if err != nil {
+		h.handleError(w, http.StatusInternalServerError, err)
+
+		return
+	}
+
+	err = json.NewEncoder(w).Encode(map[string]string{"id": feedbackID})
+	if err != nil {
+		h.handleError(w, http.StatusInternalServerError, err)
+
+		return
+	}
+
+	w.WriteHeader(http.StatusCreated)
+}
+
+// GetPageFeedbacks GET /p-feedbacks.
+func (h *Handlers) GetPageFeedbacks(w http.ResponseWriter, r *http.Request) {
+	var (
+		limit      int
+		nextInput  string
+		nextOutput string
+		nextURL    string
+		err        error
+		feedbacks  []*models.Feedback
+	)
+
+	limit, nextInput, err = validatePaginator(r.URL.Query())
+	if err != nil {
+		h.handleError(w, http.StatusBadRequest, err)
+
+		return
+	}
+
+	feedbacks, nextOutput, err = h.feedbackService.GetPage(limit, nextInput)
+	if err != nil {
+		h.handleError(w, http.StatusBadRequest, err)
+
+		return
+	}
+
+	if len(feedbacks) == 0 {
+		err = fmt.Errorf("next '%s': %w", nextInput, errNoValuesNext)
+		h.handleError(w, http.StatusBadRequest, err)
+
+		return
+	}
+
+	nextURL = fmt.Sprintf("/p-feedbacks?%s=%d&%s=%s", limitQueryParam, limit, nextQueryParam, nextOutput)
+	w.Header().Set("URL-cursor-next", nextURL)
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	err = json.NewEncoder(w).Encode(feedbacks)
+	if err != nil {
+		h.handleError(w, http.StatusInternalServerError, err)
+
+		return
+	}
+}
+
+func validatePaginator(queryParams url.Values) (int, string, error) {
+	var (
+		err   error
+		limit = defaultLimit //nolint:ineffassign
```

Comment:
never disable `ineffassign`

```suggestion
		limit int
```